### PR TITLE
[Feature][Angular] improve docs angular import

### LINKED ISSUE #7354: load two swagger generated APIs in angular 5 / Typscript project
##### Description

We have exported with Swagger 2 APIS: 

1. typescript-angular-client-generated.zip (UsersService)
2. typescript-angular-client-generated.zip (DataService)

API-1: http://example-myapi.de:8081/v2 UsersService
API-2: http://example-myapi.de:8082/v2 DataService

We try to import that...but we can't load 2 api ...UsersService and DataService.

`
 constructor(
    private usersservice: UsersService,
    private dataservice: DataService
){}
`
no Error when load one 
`
 constructor(
    private usersservice: UsersService,
    // private dataservice: DataService
){}
`
or `
 constructor(
    // private usersservice: UsersService,
    private dataservice: DataService
){}
`

##### Error message

ERROR Error: Uncaught (in promise): Error: StaticInjectorError[DataService]: 
  StaticInjectorError[DataService]: 
    NullInjectorError: No provider for DataService!
Error: StaticInjectorError[DataService]: 
  StaticInjectorError[DataService]: 
    NullInjectorError: No provider for DataService!

##### Swagger-codegen version

2.3.0 with language typescript-angular

**How can I use two apis exports in one project. Thx a lot for helping**

#####  Swagger files

[typescript-angular-client-generated(UserService).zip](https://github.com/swagger-api/swagger-codegen/files/1617917/typescript-angular-client-generated.UserService.zip)
[typescript-angular-client-generated(DataService).zip](https://github.com/swagger-api/swagger-codegen/files/1617918/typescript-angular-client-generated.DataService.zip)

### PR DESCRIPTION
**Pull request by macjohnny**
### PR checklist

- [x] Read the [contribution guidelines](https://github.com/swagger-api/swagger-codegen/blob/master/CONTRIBUTING.md).
- [x] Ran the shell script under `./bin/` to update Petstore sample so that CIs can verify the change. (For instance, only need to run `./bin/{LANG}-petstore.sh` and `./bin/security/{LANG}-petstore.sh` if updating the {LANG} (e.g. php, ruby, python, etc) code generator or {LANG} client's mustache templates). Windows batch files can be found in `.\bin\windows\`.
- [x] Filed the PR against the correct branch: `3.0.0` branch for changes related to OpenAPI spec 3.0. Default: `master`.
- [x] Copied the [technical committee](https://github.com/swagger-api/swagger-codegen/#swagger-codegen-technical-committee) to review the pull request if your PR is targeting a particular programming language.

### Description of the PR

fixes #7354

- improves generated README.md on how to use multiple generated ApiModules
- generate README.md also for non-npm packages
- remove import of `HttpModule` / `HttpClientModule` in the `ApiModule` and verify `Http` / `HttpClient` is provided by `AppModule`. See https://github.com/angular/angular/issues/20575

### CODE REVIEW

**Comment by macjohnny** [comment]
cc @JohannesHoppe

File: modules/swagger-codegen/src/main/resources/typescript-angular/api.module.mustache [hunk 1]
```diff
@@ -15,7 +10,7 @@ import { {{classname}} } from './{{importPath}}';
 {{/apiInfo}}
 
 @NgModule({
-  imports:      [ CommonModule, {{#useHttpClient}}HttpClientModule{{/useHttpClient}}{{^useHttpClient}}HttpModule{{/useHttpClient}} ],
+  imports:      [],
```
Comment (JohannesHoppe) [inline modules/swagger-codegen/src/main/resources/typescript-angular/api.module.mustache]: perfect! 👍

**Comment by JohannesHoppe** [comment]
Excellent job, I will this soon with my use-case.

**Comment by JohannesHoppe** [comment]
A great improvement. Works perfectly with my use-case.

I had the following situation:
* AppModule imports ApiModule
* ApiModule imports HttpClientModule 
.. which works great but resets the `HTTP_INTERCEPTORS`. I can confirm that your PR fixes https://github.com/angular/angular/issues/20575 when using swagger-codegen

**Comment by macjohnny** [comment]
would anyone of the technical comitee mind please reviewing the code?

cc @TiFu @taxpon @sebastianhaas @kenisteward @Vrolijkx @wing328 @bedag-moo

**Comment by wing328** [comment]
@macjohnny thanks for the enhancement to the README, which has been merged into 2.4.0

Thanks @JohannesHoppe @bedag-moo for reviewing the change.

**Pull request merged**